dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -239,8 +239,36 @@ RUN for iter in {1..10}; do \
     (exit $exit_code)
 {{- end }}
 
-USER {{ .user }}
+{{- if (and (eq .Variant "wolfi-complete") (contains .from "wolfi"))  }}
+USER root
+# Install required dependencies from wolfi repository
+RUN for iter in {1..10}; do \
+        apk update && \
+        apk add --no-interactive --no-progress --no-cache nodejs-18=18.20.4-r0 npm=10.8.3-r0 glib dbus-glib libatk-1.0 \
```

Comment:
can you update the NODE_VERSION on ubuntu as wel - https://github.com/elastic/elastic-agent/blob/cec25d6e8fb0b76f5a50e78d022d4fcffad26287/dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl#L195

Reply:
done in https://github.com/elastic/elastic-agent/pull/5475/commits/6d9ad70957ae24046997911cfbecf004dca8f0cf